packages/components/src/InputAvatar/InputAvatar.tsx

```diff
@@ -52,7 +58,7 @@ export function InputAvatar({
         getUploadParams={getUploadParams}
         onUploadStart={handleChange}
         onUploadProgress={handleUpload}
-        onUploadComplete={handleUpload}
+        onUploadComplete={handleUploadComplete}
```

Comment:
It didn't take long for us to find a use case for having split these events out, so I guess that was a good decision. :)